src/PKSim.Infrastructure/Services/PKSimLoggerCreator.cs

```diff
@@ -0,0 +1,44 @@
+﻿using PKSim.Infrastructure.Services;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Console;
+using NHibernate.Mapping;
```

Comment:
not sure why this is using NHiberate, Console etc..

Reply:
@msevestre @georgeDaskalakis I have just removed those. We do not need them.